barberia/src/App/pages/Home.js

```diff
@@ -1,19 +1,17 @@
 import React, { Component } from "react";
 import { Link } from "react-router-dom";
-
+import {routes} from "../App";
+import {A} from "hookrouter";
 class Home extends Component {
```

Comment:
Any particular reason why this is a Class based Component? 

You've used Functional Components elsewhere, and this is just returning JSX (ie: no state). I'd advice you change this to be a function.

Reply:
refactored